index.html

```diff
@@ -95,15 +95,15 @@ <h2 class="intro-info2">2023.11.27(Mon) ~ 30(Thur)</h2>
 
 
   <section class="main-program flex-column-center" id="main-program">
-    <h3 class="text-center text-white mt-4 opacity-75">Main Program</h3>
+    <h3 class="text-center text-white">Main Program</h3>
       <div class="underline-bottom opacity-75"></div>
     <div class="program-items">
       <article class="program-item flex-row-center bg-light-black">
-        <div class="item item-icon heading-big">
+        <div class="item item-icon">
           <img src="image/program_icon_01.png" class="img-fluid rounded-start" alt="...">
         </div>
         <h3 class="item item-header dark-orange">Lecture</h3>
```

Comment:
- Kindly improve the design of the headline section as requested by the previous reviewer :recycle: , For this I suggest:
  - Making sure it covers the whole width of the screen :recycle: 
  - Align the content(text) properly as per the design:

Your design:
![Screenshot from 2023-04-23 00-14-47](https://user-images.githubusercontent.com/97834160/233808840-b28e9314-4d1e-4e1c-9f78-65f4703596ce.png)
Cindy's design:

![Screenshot from 2023-04-23 00-15-53](https://user-images.githubusercontent.com/97834160/233808871-c81e046e-b954-4e3c-a4a6-3110461a5193.png)